StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSession.swift

```diff
@@ -122,7 +152,9 @@ public extension StripeAPI {
                 accounts: accounts,
                 livemode: try container.decode(Bool.self, forKey: .livemode),
                 paymentAccount: try? container.decode(PaymentAccount.self, forKey: .paymentAccount),
-                bankAccountToken: try? container.decode(BankAccountToken.self, forKey: .bankAccountToken)
+                bankAccountToken: try? container.decode(BankAccountToken.self, forKey: .bankAccountToken),
+                status: (try? container.decode(Status.self, forKey: .status)) ?? .pending,
```

Comment:
@kgaidis-stripe it seems semantically more correct to have the default value as `.unparsable` as it is indeed unparsable in this case.